src/main/scala/org/apache/spark/sql/pulsar/PulsarConfigUpdater.scala

```diff
@@ -76,4 +81,14 @@ private[pulsar] case class PulsarConfigUpdater(
     map
   }
 
+  private val HIDE_COMPLETELY_LIMIT = 6
```

Comment:
This is not a static field, so just use camel case by following scala style.

Reply:
Done.